src/Components/Input/input.jsx

```diff
@@ -1,5 +1,23 @@
 import "./input.scss";
 
-export default function Input({ placeholder, name, type = "text" }) {
-  return <input type={type} name={name} id={name} placeholder={placeholder} />;
+export default function Input({
+  placeholder,
+  name,
+  type = "text",
+  value,
+  setInputState,
+}) {
+  function handleChange(event) {
+    setInputState(event.target.value);
+  }

```

Comment:
```suggestion
  }

```

Add blank line after closed the function